src/test/groovy/de/eddyson/tapestry/react/integration/AlertDemoSpec.groovy

```diff
@@ -24,7 +24,9 @@ class AlertDemoSpec extends JettyGebSpec {
     then:
     // TODO: this will need a custom ApplicationStatePersistenceStrategy
     // applicationStateManager.get(AlertStorage).alerts.size() == 1
-    helloWorld.displayed
+    waitFor {
```

Comment:
But that wouldn't happen without the change to `src/test/resources/de/eddyson/testapp/pages/ReactDemo.tml`, right? In the end, I'd like you to revert that too. ;-)